app/src/main/java/org/oppia/app/profile/AdminPinViewModel.kt

```diff
@@ -1,13 +1,15 @@
 package org.oppia.app.profile
 
-import androidx.databinding.ObservableField
+import androidx.lifecycle.MutableLiveData
 import org.oppia.app.activity.ActivityScope
 import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 /** The ViewModel for [AdminPinActivity]. */
 @ActivityScope
 class AdminPinViewModel @Inject constructor() : ObservableViewModel() {
-  val pinErrorMsg = ObservableField("")
-  val confirmPinErrorMsg = ObservableField("")
+  val pinErrorMsg = MutableLiveData("")
+  val confirmPinErrorMsg = MutableLiveData("")
+  val savedPin = MutableLiveData("")
+  val savedConfirmPin = MutableLiveData("")
```

Comment:
The concept that you had mentioned earlier that `MultableLiveData` saves data across orientation change.  Actually the data persists across orientation change because of ViewModel and not `MutableLiveData`. At the same time , `MutableLiveData` as well as `ObservaleField` both are are correct approach. For now as this is also right we can approve this but in future PRs do use `ObservableField` so that the entire code is consitent in app as all other ViewModels use `ObservableField` and not `MutableLiveData`

Reply:
I had previously tried with `ObservableField("")` before using `MutableLiveData("")` but it was not saving the state. 😓 Maybe I had done something wrong. Thanks for your feedback.